src/deluge/processing/sound/sound.cpp

```diff
@@ -1253,6 +1253,11 @@ Error Sound::readTagFromFile(StorageManager& bdsm, char const* tagName, ParamMan
 		bdsm.exitTag("polyphonic");
 	}
 
+	else if (!strcmp(tagName, "maxVoices")) {
+		maxVoiceCount = bdsm.readTagOrAttributeValueInt();
+		bdsm.exitTag("maxVoices");
+	}
+
```

Comment:
Does this cause official to crash/not load subsequent settings?

Reply:
yep, moved to the end